Loggie/Classes/LogDetails/LogDetailsViewController.swift

```diff
@@ -130,3 +130,59 @@ extension LogDetailsViewController: UITableViewDataSource {
         }
     }
 }
+
+extension LogDetailsViewController: UITableViewDelegate {
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let actionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        actionSheet.addAction(copyAction(for: indexPath))
+        actionSheet.addAction(shareAction(for: indexPath))
+
+        present(actionSheet, animated: true)
+    }
+
+    private func copyAction(for indexPath: IndexPath) -> UIAlertAction {
```

Comment:
In newer versions of Swift you can create private extensions and group this methods there, for example:

```swift
private extension LogDetailsViewController {
    func somePrivateMethod() {
    }
}
```

Reply:
Done.